Jint.Tests.PublicInterface/Jint.Tests.PublicInterface.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
```

Comment:
please use the same dependencies as in Jint.Tests.csproj:

```xml
  <ItemGroup>
    <PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" PrivateAssets="all" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5" />
  </ItemGroup>
```

Reply:
Resolved, thanks.  I cloned the Jint.Tests.csproj file instead of using the default content that VS generated.